apollo-gradle-plugin/src/test/testProject/android/buildScriptFixtures/flavored.gradle

```diff
@@ -0,0 +1,60 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:2.3.0-beta3'
+  }
+}
+
+plugins {
+  id 'com.android.application'
+  id 'com.apollographql.android'
+}
+
+android {
+  compileSdkVersion 25
+  buildToolsVersion '25.0.2'
```

Comment:
These should be in a properties file. I'm assuming as we update version of sample we would want tests to update too.